library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt

```diff
@@ -978,7 +978,7 @@ data class TvSeriesSearchResponse(
     override var type: TvType? = null,
 
     override var posterUrl: String? = null,
-    val year: Int? = null,
+    var year: Int? = null,
     val episodes: Int? = null,
```

Comment:
If you change year, then also change episodes to a var.

Reply:
I can if you want but I am kinda confused why?
Year needs to be so it can be changed when using newTVSeriesSearchResponse but with that you can pass episodes.

Reply:
Nevermind I just realized I was looking at newTVSeriesLoadResponse... I will change this. Sorry